onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc

```diff
@@ -93,7 +93,8 @@ class NhwcInferenceContext : public InferenceContext {
   const SparseTensorProto* getInputSparseData(size_t) const override {
```

Comment:
No changes in this file ?

Reply:
It was originally changed, however, I merged with master where the same changes were made by Negin and this is the leftover.